src/Transfer/Resources/Auth/Hash.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Auth;
+
+use Utopia\Transfer\Resource;
+use Utopia\Transfer\Transfer;
+
+/**
+ * Helper class for hashing.
+ */
+class Hash extends Resource
+{
+    public const SCRYPT_MODIFIED = 'ScryptModified';
```

Comment:
lets prefix with `ALGORITHM_` and make sure the value is camel case or snake case